src/main/java/homeTry/exerciseList/service/ExerciseService.java

```diff
@@ -31,7 +31,10 @@ public ExerciseService(ExerciseRepository exerciseRepository,
     public void createExercise(ExerciseRequest request, MemberDTO memberDTO) {
         Member foundMember = memberService.getMemberEntity(memberDTO.id());
         Exercise exercise = new Exercise(request.exerciseName(), foundMember);
+        ExerciseTime currentExerciseTime = new ExerciseTime(exercise);
+
         exerciseRepository.save(exercise);
+        exerciseTimeService.saveExerciseTime(currentExerciseTime);
     }
```

Comment:
```suggestion
        Exercise exercise = new Exercise(request.exerciseName(), foundMember);
        exerciseRepository.save(exercise);
        
        ExerciseTime currentExerciseTime = new ExerciseTime(exercise);
        exerciseTimeService.saveExerciseTime(currentExerciseTime);
    }
```

처럼 배치하는 것이 같은 관심사끼리 배치하는 방법이라고 생각해요.
이렇게 되면,
1. 운동 저장
2. 운동을 바탕으로, 운동 시간 저장
의 흐름이 더 잘 이해가 될 것 같아요.

Reply:
참고로 말씀드리면, 채팅을 구현하면서 pub-sub 구조와 이벤트에 대해 공부해보셨을 테니,, 고민해보실만한 내용 하나 전달드려요.
createExercise 메서드는 사실 "운동을 생성(저장)" 을 담당하는 메서드에요.
운동 시간을 저장하는 것까지가 관심사일까? 에 대한 고민이 생길텐데요.
이렇게 관심사를 분리하는 것도 이벤트로 빼낼수 있어요.
예를 들면,, ExerciseEventPublisher.publishCreationEvent(ExerciseCreationEvent);  와 같은 메서드만 작성해서,
별도의 listener에서 운동이 생성되었을 때 발생하는 이벤트(여기서는 운동시간 기록, 다른 일도 추가될 수 있을 것 같아요..)를 분리해서 핸들링할 수 있습니다. 참고만 해주세요~